spec/lcm/load/users_brick_load_spec.rb

```diff
@@ -121,5 +139,58 @@ def user_in_domain(user_name)
         expect(this_project.member?(u)).to be_truthy unless this_project.nil?
       end
     end
+
+    it 'sets the right MUFs to right users' do
+      $SCRIPT_PARAMS = JSON.parse(File.read(@config_path))
+      @test_context = {
+        project_id: @project.pid,
+        config: LcmConnectionHelper.environment,
+        s3_bucket: Support::S3Helper::BUCKET_NAME,
+        s3_endpoint: Support::S3Helper::S3_ENDPOINT,
+        s3_key: 'user_data',
+        users_brick_input: {
+          s3_bucket: Support::S3Helper::BUCKET_NAME,
+          s3_endpoint: Support::S3Helper::S3_ENDPOINT,
+          s3_key: 'users_brick_input'
+        }
+      }
+      @ads = GoodData::DataWarehouse.create(
+        client: @rest_client,
+        title: 'TEST ADS',
+        auth_token: LcmConnectionHelper.environment[:prod_token]
+      )
+      @test_context[:jdbc_url] = @ads.data['connectionUrl']
+      @ads_client = GoodData::Datawarehouse.new(
+        @test_context[:config][:username],
+        @test_context[:config][:password],
+        nil,
+        jdbc_url: @ads.data['connectionUrl']
+      )
+      ads_template_path = File.expand_path(
```

Comment:
warning: Lint/UselessAssignment: Useless assignment to variable - `ads_template_path`. (rubocop)